src/main/webapp/app/modeling-assessment-editor/modeling-assessment-conflict/modeling-assessment-conflict.component.ts

```diff
@@ -203,17 +202,35 @@ export class ModelingAssessmentConflictComponent implements OnInit, AfterViewIni
     private updateHighlightColor() {
         switch (this.conflictResolutionStates[this.conflictIndex]) {
             case ConflictResolutionState.UNHANDLED:
-                this.highlightColor = 'rgba(0, 123, 255, 0.6)';
+                this.setHighlightColorOfConflictElements(FeedbackHighlightColor.BLUE);
                 break;
             case ConflictResolutionState.ESCALATED:
-                this.highlightColor = 'rgba(255, 193, 7, 0.6)';
+                this.setHighlightColorOfConflictElements(FeedbackHighlightColor.YELLOW);
                 break;
             case ConflictResolutionState.RESOLVED:
-                this.highlightColor = 'rgba(40, 167, 69, 0.6)';
+                this.setHighlightColorOfConflictElements(FeedbackHighlightColor.GREEN);
                 break;
         }
     }
 
+    private setHighlightColorOfConflictElements(color: string) {
+        if (this.conflictingModelHighlightedElements) {
```

Comment:
You could think about initializing the variable with an empty `Map` so you can reduce complexity a bit.

Reply:
Good point, I changed it.